apps/chat/src/Config.js

```diff
@@ -5,12 +5,12 @@ import data from './backend/serverless/appconfig.json'
 const appConfigJson = Object.assign({}, ...data.map((x) => ({[x.OutputKey]: x.OutputValue})));
 
 const appConfig = {
-    apiGatewayInvokeUrl: '' || appConfigJson.apiGatewayInvokeUrl,
-    cognitoUserPoolId: '' || appConfigJson.cognitoUserPoolId,
-    cognitoAppClientId: '' || appConfigJson.cognitoAppClientId,
-    cognitoIdentityPoolId: '' || appConfigJson.cognitoIdentityPoolId,
-    appInstanceArn: '' || appConfigJson.appInstanceArn,
+    apiGatewayInvokeUrl: appConfigJson.apiGatewayInvokeUrl,
```

Comment:
Do we want to leave the ||?  Or think certain directions/blogs say to update in this file

Reply:
specifically concerned about: https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#running-the-chat-demo-app-locally

Reply:
yea, better to keep it as the ReadMe mentions to update these params manually if the stack is deployed manually. updated.